helpers/handlebars.js

```diff
@@ -0,0 +1,76 @@
+const handelbarsHelpers = {
+  not: (v) => {
```

Comment:
nitpickings:
* useless parens around `v`
* you can use arrow functions implicit return to simplify the syntax

Example:

```javascript
{
  not: v => !v,
  eq: (v1, v2) => v1 === v2,
  ne: (v1, v2) => v1 !== v2,
  ...
```